db/models/review.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+const {STRING, TEXT, ENUM} = require('sequelize')
+
+module.exports = db => db.define('review', {
+    title: {
+        type: STRING,
+        allowNull: false
+    },
+    content: {
+        type: TEXT,
+        validate: {
+            notNull: true,
+            len: [100, 2800]
+            }
+    },
+    rating: {
+        type: ENUM(1,2,3,4,5)
```

Comment:
should be integer with a range so that we can more easily order them if we so choose.